Minor fixes for 2.0.0 (#148)
aerospike/kubernetes-operator: docs/Aerospike-access-control.md

description: Aerospike Access Control
---

> Aerospike Access Control includes user, role, and privilege creation and maintenance. [See the documentation for more information on Aerospike Access Control](https://docs.aerospike.com/docs/operations/configure/security/access-control/).

Reviewer: ```suggestion
Aerospike Access Control includes user, role, and privilege creation and maintenance. [See the documentation for more information on Aerospike Access Control](https://docs.aerospike.com/docs/operations/configure/security/access-control/index.html).
```

Reviewer: our current docs system requires index.html in order to select the correct thing in the left hand nav. In our new docs that won't be an issue but once new docs is up we will probably have to audit our links

To manage your access controls from the operator, configure the `aerospikeAccessControl` section in the Aerospike cluster's Custom Resource (CR) file.

:::warning

> Access control changes on an operator managed Aerospike cluster made externally, e.g. using AQL or Asadm, will be reverted by the operator.

Reviewer: ```suggestion
Access control changes on an operator-managed Aerospike cluster will be reverted if made made externally (e.g. using `aql` or `asadm`).
```
